packages/x-tree-view/src/internals/useTreeView/extractPluginParamsFromProps.ts

```diff
@@ -54,10 +54,16 @@ export const extractPluginParamsFromProps = <
     }
   });
 
+  const cleanExperimentalFeatures =
```

Comment:
Super nitpick: `cleanExperimentalFeatures` sounds a bit weird.. I get why it's called that, and I also don't have a better idea for what to call it, so I guess it's fine. 😅

Reply:
Changed from `experimentalFeatures` => `cleanExperimentalFeatures` to `inExperimentalFeatures` => `experimentalFeatures`